serviceerror/system_workflow.go

```diff
@@ -27,6 +27,14 @@ func NewSystemWorkflow(workflowExecution *common.WorkflowExecution, workflowErro
 	}
 }
 
+// NewSystemWorkflowf returns new SystemWorkflow error with formatted workflow error.
+func NewSystemWorkflowf(workflowExecution *common.WorkflowExecution, format string, args ...interface{}) error {
```

Comment:
I can't imagine ever using this, it should always be constructed from an `error`